docs/admin/config/batch_changes.mdx

```diff
@@ -344,12 +344,37 @@ However, if the user deletion is permanent, deleting both account and data, then
 
 ## Batch Spec Library
 
-<Callout type="note"> Batch Spec Library is currently in Experimental.</Callout>
+The Batch Spec Library accelerates batch change adoption by offering curated examples and guided templates that make large-scale code modifications accessible to developers at every skill level.
```

Comment:
To me, this sounds like it should be internal because of the word "adoption" -- like, it seems like we're saying leaving a note for our marketing team that the Library aims to increase BC adoptions b/c XYZ. Maybe rephrase to something like "The Batch Spec Library offers curated examples..." ?